src/criteria.py

```diff
@@ -0,0 +1,51 @@
+from symptom import Symptom
+from signal_statistics import SignalStatistics
+import constants
+
+""" Module for defining criteria to be used to evaluate event groups"""
+
+class Criteria:
+    """ Object for detecting and evaluating signals resulting from
+        event groups. Given some symptom tag, a signal to detect,
+        and an evaluator for that signal, the criteria will detect
+        and build instances of these symptoms
+
+        Attributes
+        ----------
+        symptom_tag: string name for the symptom this criteria defines
+        signal: signal object to detect
+        evaluator: evaluator object to apply to detected signals
+        action_msg: string message to build into a detcted symptom
+
+    """
+
+    def __init__(self, symptom_tag, signal, evaluator, action_msg):
+        self.symptom_tag = symptom_tag
+        self.signal = signal
+        self.evaluator = evaluator
+        self.action_msg = action_msg
+
+    def apply_criteria(self, event_groups, collect_statistics):
```

Comment:
Please add docstrings for all of the functions here as well, unless they're completely trivial.